projects/step-frontend/src/lib/modules/timeseries/performance-view/table/timeseries-table.component.scss

```diff
@@ -9,3 +9,11 @@
     margin-right: 12px;
   }
 }
+
+.ts-table {
+  ::ng-deep {
```

Comment:
As I know `::ng-ddep` is deprecated.
https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep
It is better to avoid it's usage.

For your case - is it possible to add a class to the cell and specify the cell's height?
In that case row's height will align to cell's height.

Reply:
In case ::ng-deep needs to be used encapsulated it with :host, but I agree with @dvladir that we should try to avoid it.